### DISCUSSION TITLE
Fixed Date Buttons

### DISCUSSION THREAD

**Discussion by rsidhaarth** [Blair2004/NexoPOS#399] [Ideas]
Hi Team,

I notice that in many places, we have been just provided with the option to choose a date range our self. But you can provide these buttons by default

1) Today
2) Yesterday
3) This Week
4) Last Week
5) This Month
6) Last Month
7) Past 7 Days
8) Past 14 Days
9)  Past 30 Days

These can be used on the dashboard, reports area

Whereas for Kitchen Screen, I notice that you have also asked us to choose a date range up to which time we need notification. Instead, you can provide us with the below option.

Live Updates once in 

1) 5 seconds
2) 10 seconds
3) 15 seconds
4) xx seconds
5) xx minutes

so that, the screen will get live updates about the order in respective seconds or minutes. And also an alarm is required as people won't keep seeing it.

**Comment by Blair2004** [comment]
Hi, 
That's a nice improvement suggestion. We'll keep it aside from now.
Thanks.
